tests/integration_tests/databases/api_tests.py

```diff
@@ -2850,6 +2850,136 @@ def test_function_names(self, mock_get_function_names):
         assert rv.status_code == 200
         assert response == {"function_names": ["AVG", "MAX", "SUM"]}
 
+    def test_function_names_sqlite(self):
+        example_db = get_example_database()
+        if example_db.backend != "sqlite":
+            return
+
+        self.login(username="admin")
+        uri = "api/v1/database/1/function_names/"
+
+        rv = self.client.get(uri)
+        response = json.loads(rv.data.decode("utf-8"))
+
+        assert rv.status_code == 200
+        assert response == {
+            "function_names": [
+                "abs",
```

Comment:
wdyt about defining this list as a separate variable in the db engine spec and then importing it here, or do you want this to be explicitly updated here when the original list is updated?

Reply:
Yeah, I tend to avoid comparing results to imported variables in unit tests, since it can lead to mistakes.